RELEASING/release-notes-2-0/changelog.md

```diff
@@ -128,6 +128,7 @@ under the License.
 - [#19408](https://github.com/apache/superset/pull/19408) feat(dashboard): Implement empty states for empty tabs (@kgabryje)
 - [#19446](https://github.com/apache/superset/pull/19446) feat(explore): Move chart actions into dropdown (@kgabryje)
 - [#19394](https://github.com/apache/superset/pull/19394) feat(explore): UI changes in dataset panel on Explore page (@kgabryje)
+- [#24172](https://github.com/apache/superset/pull/24172) feat: add connector for QuestDB timeseries database (@marregui)
```

Comment:
@marregui you're updating the changelog of a release that has already been shipped. The only place where we record future changes is in [UPDATING.md](https://github.com/apache/superset/blob/master/UPDATING.md), however this is used for breaking changes, which isn't the case for your PR.

Reply:
@marregui please add the dependency to `requirements/testing.in` [here](https://github.com/apache/superset/blob/master/requirements/testing.in#L19) and then you'll have to re-compile the frozen dependencies to ensure that CI is deterministic.